src/blocks/pagination/Pagination.types.ts

```diff
@@ -0,0 +1,12 @@
+export type PaginationProps = {
+  //onChange function
+  onChange: (page: number) => void;
+  // items per page
+  perPage: number;
+  // startPage of pagination
+  startPage: number;
```

Comment:
This should be `current` and not `startPage`. Because this is going to change as user clicks on any page number so current makes more sense than startPage.